NitroxInstallerActions/Patches/NitroxServerSetup.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace InstallerActions.Patches
+{
+    public static class NitroxServerSetup
+    {
+        public static void Init(string dir)
+        {
+            if (File.Exists(Path.Combine(Path.GetFullPath(dir), "..\\..\\SubServer\\Assembly-CSharp.dll")))
```

Comment:
For both readability, and maintainability I would suggest creating variables for the Path.Combine results, and constants for the relative paths.